planner/cascades/testdata/transformation_rules_suite_in.json

```diff
@@ -112,6 +112,12 @@
       "select a from (select a from t where b > 2 order by a, b limit 3 offset 1) as t1 order by a limit 2 offset 1"
     ]
   },
+  {
+    "name": "TestMergeAdjacentWindow",
+    "cases": [
+      "select sum(a) over(partition by b order by c), max(b) over(partition by b order by c) from t"
```

Comment:
Please add some cases for the comparison of other fields, such as `WindowFrame`.